### PR TITLE
Update Onboarding Mentorship Documentation

### PR DESCRIPTION
**Pull request by iskounen**
This PR updates documentation related to mentorship that appears in 3 locations:
1. Onboarding checklist for managers
2. Mentoryship at Artsy doc
3. Sprint rotation doc

### CODE REVIEW

**Comment by sweir27** [comment]
TODOs from our meeting:
- [ ] Align with @joeyAghion on clarifying expectations for the rotation buddy
- [ ] Clarify lunch expectation-- maybe suggest a group walks to Canal st. market to bring food back (but that the manager buys the hew hire's lunch?)

File: onboarding/managers.md [hunk 1]
```diff
@@ -5,21 +5,32 @@ description: Things for managers to remember during the early days of onboarding
...
 - [ ] Create [an onboarding github issue](https://github.com/artsy/README/issues/new?template=engineering-onboarding.md) in this repo based on the onboarding template. Also invite the team to say hi, as this is a place that the new hire should always be able to find easily in the future.
 - [ ] Set first-week schedule and let the new hire know their first day schedule on their private email account, as there will be a period where their machine is still being setup and they won’t yet have access to their calendar.
 - [ ] Invite new hire to all engineering-wide meetings
+- [ ] Invite new hire to a 1:1 on day 1
 - [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md)
+- [ ] Remind tech leads/PMs that are part of the sprint rotation to prepare resources and invite new hire to meetings
+- [ ] Familiarize yourself with people resources across the company that will be important to your mentee:
+  - Point people in Workplace & People Ops
+  - Their manager
+  - Their team lead
```
Comment (sweir27) [inline onboarding/managers.md]: I'm not sure what this means in this context (they don't have a "team" yet necessarily)

File: onboarding/managers.md [hunk 2]
```diff
@@ -5,21 +5,32 @@ description: Things for managers to remember during the early days of onboarding
...
 
 - [ ] Choose a peer mentor for the new hire
 - [ ] Choose a spirit guide for the new hire (consult with People Ops)
 - [ ] Create [an onboarding github issue](https://github.com/artsy/README/issues/new?template=engineering-onboarding.md) in this repo based on the onboarding template. Also invite the team to say hi, as this is a place that the new hire should always be able to find easily in the future.
 - [ ] Set first-week schedule and let the new hire know their first day schedule on their private email account, as there will be a period where their machine is still being setup and they won’t yet have access to their calendar.
 - [ ] Invite new hire to all engineering-wide meetings
+- [ ] Invite new hire to a 1:1 on day 1
 - [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md)
+- [ ] Remind tech leads/PMs that are part of the sprint rotation to prepare resources and invite new hire to meetings
+- [ ] Familiarize yourself with people resources across the company that will be important to your mentee:
```
Comment (sweir27) [inline onboarding/managers.md]: You may want to re-work this section a little bit so it pertains to the manager

File: onboarding/managers.md [hunk 3]
```diff
@@ -5,21 +5,32 @@ description: Things for managers to remember during the early days of onboarding
...
+  - Their team lead
+  - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
 
 ### Day 1
 
 - [ ] Organize lunch outing (probably to Tataki!)
 - [ ] Give an overview of what the onboarding will look like. This is a good time to walk through the onboarding template.
 - [ ] Introduce new hire to their mentor
 - [ ] When they are set up with Slack add them to the `@developers` [user group](https://artsy.slack.com/admin/user_groups) 🔒, which should automatically invite them to developer related channels, and invite them to channels relavent to the individual
+- [ ] When they are set up with an Artsy email address add them to the developers email group
```
Comment (sweir27) [inline onboarding/managers.md]: I believe that this is handled by IT, but we can verify with Jason

File: onboarding/mentors.md [hunk 4]
```diff
@@ -25,16 +25,14 @@ It's up to the mentoring pair to find what works best for them and to ensure the
...
   - Introduce yourself
   - Set a time for an [introductory coffee or tea](#your-first-meeting)
-  - Get in touch with the mentee's manager
-  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
-    - Point people in Workplace & People Ops
-    - Their manager
-    - Their team lead
-    - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
+  - Introduce the engineering team activities (Monday stand-up, Lunch & Learn, Demo Days, etc.)
+  - Introduce the on-call process
```
Comment (sweir27) [inline onboarding/mentors.md]: There's a separate, more in-depth version of this as part of the onboarding checklist. Is the goal of this item to just let the person know that there _is_ an on-call process? (If so we should be specific 😄 )

File: onboarding/mentors.md [hunk 5]
```diff
@@ -45,9 +43,15 @@ Get to know each other. This can be a nice, casual conversation. Focus on active
...
 - Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+  - Be available to troubleshoot checklist items.
+  - Encourage (pair) on documenting problems with the checklist.
 - Look for chances to pair or work alongside one another throughout the week.
   - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
   - If they haven't set up hokusai yet, consider pairing on that.
+  - Notice what types of tasks they are good at and connect them to engineers who share that strength. (https://dev.to/flippedcoding/how-to-be-a-good-software-engineer-mentor-4b)
+  - Notice what types of tasks they struggle with and connect them to engineers who can help. (https://dev.to/flippedcoding/how-to-be-a-good-software-engineer-mentor-4b)
+  - Encourage improvements to READMEs while setting up projects.
+- Take your onboardee to team meetings and explain any business context after attending.
```
Comment (sweir27) [inline onboarding/mentors.md]: This is a good place to be explicit about expectations if you're remote!

i.e. "Take your onboardee to team meetings and explain any business context after attending. If you're in the office, drop by their desk and bring them in-person. If you're remote, make a point to slack them to remind." (or something like that)

File: onboarding/sprint-rotation.md [hunk 6]
```diff
@@ -30,6 +30,14 @@ Following the sprint rotation, the new engineer will join a team full-time.
   - Schedule of tech leads/PMs
   - Specific areas of interest for new engineer
 - Manager makes sure the tech lead/PM are aware (well in advance) of the upcoming rotation.
+- Manager selects a rotation buddy.
+
+##### Rotation buddy
+
+The goal of the rotation buddy is make the onboardee feel comfortable on their rotation team and provide context and information about ongoing projects. The rotation buddy may be their mentor for one rotation, but the goal is not to provide mentorship. Responsibilities include:
```
Comment (sweir27) [inline onboarding/sprint-rotation.md]: I think the "the goal is not to provide mentorship" is slightly misleading! It seems clear enough without that caveat.

File: onboarding/mentors.md (hunk 4 above)
Comment (iskounen) [reply onboarding/mentors.md]: The checklist's section on this subject is perfect. I'll remove this.

**Comment by iskounen** [comment]
Some more TODOs from our meeting:

- [ ] Manager needs to be aware of all of the setup that needs to be happen, and who is responsible for doing it. This includes things like AWS access, GItHub permissions, OpsGenie setup, etc.
- [ ] Double-check the manager/mentor/rotation buddy documentation to ensure that it is remote-friendly.
- [ ] Mentors should be able to identify bugs worth fixing in Sentry for the first task during week 1.

**Comment by starsirius** [comment]
@iskounen 👍 to the TODO about setting up access to AWS, Github, etc. before day 1. I also added corresponding note in the onboarding [checklist](https://github.com/artsy/potential/pull/304/files#diff-f568dd313098d3297a83c0b0e450a0abR41).

I reached out to Jason for list of setup IT helped with, in case it's useful. This is the list:
```
* G-Suite- Gmail, Google Docs
* Microsoft Office 365
* Adobe Creative Cloud
* Zoom
* Skype
* Artsy Admin
* Google Analytics
* Looker
* Cloudapp
* Boomerang
* Github
* Slack
* Dropbox
```

File: onboarding/sprint-rotation.md (hunk 6 above)
Comment (iskounen) [reply onboarding/sprint-rotation.md]: Reading @joeyAghion's description of the rotation buddy, I think that a new section isn't necessary, so I will scrap these changes.

File: onboarding/mentors.md [hunk 7]
```diff
@@ -21,33 +21,38 @@ It's up to the mentoring pair to find what works best for them and to ensure the
 ## How to mentor
 *Moving from the abstract to the concrete:*
 
+### Before your first day
+- Become intimately familiar with the contents of the [onboarding checklist](https://github.com/artsy/potential/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md).
+- Identify issues in Sentry that are appropriate for a first task that can be pushed to production during their first week.
```
Comment (iskounen) [inline onboarding/mentors.md]: I would like to expand on this in either this doc or in a separate doc that we link to from here. @joeyAghion would you be able to help me understand how to identify these types of issues?

File: onboarding/managers.md (hunk 3 above)
Comment (ashkan18) [reply onboarding/managers.md]: yes, this is now part of IT's onboarding list, we can remove from here.

**Comment by SamRozen** [comment]
This PR has been open for a while, is it still relevant?

**Comment by iskounen** [comment]
I'd like to wrap this up. Let's keep it open.

**Comment by dblandin** [comment]
@iskounen We renamed the default branch on this repository from `master` to `main` today and were unable to keep this PR open during the process. Feel free to reopen your PR targeting the new default branch.

**Comment by kajatiger** [comment]
Yeah something is weird here. I merely renamed the default branch in the settings and automatically this PR was closed and the working branch was deleted (github says it was me, but I did not actively manually do this). My own PR on the other side was only turned into a draft PR now pointing to main. I wonder why github is making a difference here.

**Pull request closed without merging**